### DISCUSSION TITLE
How do discussions work?

### DISCUSSION THREAD

**Discussion by moonheart08** [bevyengine/bevy#507] [Q&A] [answered]
Is this just issues?

**Comment by cart** [comment, marked as answer]
It behaves a lot like issues, but there are a few key differences:

1. You can ask "questions" that can receive "answers"
2. Unlike issues, the goal isn't to "close" discussions. They can go on forever.
3. Discussions have categories
4. Discussions are "ranked" by their popularity

**Comment by steelx** [comment]
great idea.

**Comment by porkbrain** [comment]
I see it as MS's SO alternative.

**Reply by SiyahaS** [reply]
I would argue that this is an alternative. I see this as more of a feature request kind of platform. As more often than not combinations of libraries are a SO question topic.
